src/SearchAspect.php

```diff
@@ -23,4 +26,9 @@ public function getType(): string
 
         return Str::plural($type);
     }
+
+    public function limit($limit) : void
```

Comment:
```suggestion
    public function setLimit($limit) : void
```
I think it would be a good idea to use setLimit to keep consistency, see: https://github.com/spatie/laravel-searchable/blob/93a5d19aeacaf65ce55c238465c045348c557b93/src/SearchResult.php#L28